### PR TITLE
Add premium

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10103

According to MC533652

@qianw211 Qian, as the author of the license block, can you kindly review this PR? Thank you

### CODE REVIEW

File: Teams/meeting-avatars.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Avatars for Microsoft Teams lets your users connect with presence in Teams witho
 > [!NOTE]
 > The avatars for Teams app has minimum and recommended hardware requirements. For more information, see [Hardware requirements for Microsoft Teams](hardware-requirements-for-the-teams-app.md).
 >
-> Teams customers can access this feature if they have one of the following licenses: Microsoft 365 Enterprise E3, Microsoft 365 Enterprise E5, Microsoft 365 Business Standard, and Microsoft 365 Business Premium License.
+> Teams customers can access this feature if they have one of the following licenses: Microsoft 365 Enterprise E3, Microsoft 365 Enterprise E5, Microsoft 365 Business Standard, Microsoft 365 Business Premium License and Teams Premium.
```
Comment (JohanFreelancer9) [inline Teams/meeting-avatars.md:28]:
```suggestion
> Teams customers can access this feature if they have one of the following licenses: Microsoft 365 Enterprise E3, Microsoft 365 Enterprise E5, Microsoft 365 Business Standard, Microsoft 365 Business Premium License, or Teams Premium.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@DaniEASmith can you have a look at this one and merge if appropriate.

**Comment by DaniEASmith** [comment]
@MikePlumleyMSFT We already have an open PR to address this question. David and I are on a chat thread with the primary PMs, Biz Planners, and Marketing folks to come to a best way to communicate Teams Premium and Avatars as it's not a primary licensing scenario for accessing Avatars. David and I were told to hold on this until they better understand.

image

Here's a link to the other PR for reference: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/pull/11812.

**Comment by MikePlumleyMSFT** [comment]
There's a different PR in the works to address this.

**Pull request closed without merging**